controllers/admin/AdminSearchConfController.php

```diff
@@ -472,7 +472,11 @@ public function processSave()
         }
 
         if (empty($this->errors)) {
-            $this->confirmations[] = $this->trans('Creation successful', [], 'Admin.Shopparameters.Notification');
+            if (Tools::getValue('id_alias')) {
+                $this->confirmations[] = $this->trans('Successful update.', [], 'Admin.Notifications.Success');
+            } else {
+                $this->confirmations[] = $this->trans('Successful creation.', [], 'Admin.Notifications.Success');
+            }
```

Comment:
```suggestion
            if (Tools::getValue('id_alias')) {
                $this->confirmations[] = $this->trans('Update successfull', [], 'Admin.Notifications.Success');
            } else {
                $this->confirmations[] = $this->trans('Creation successful', [], 'Admin.Notifications.Success');
            }
```

Reply:
Hi @PululuK it's beacause this wording already exist at the same place ( Admin.Notifications.Success ) but i can use : 

Update successful in `Admin.Notifications.Success`

And 

Creation successful in `Admin.Shopparameters.Notification
`
The translations are not at the same place.

Reply:
Or i add a new wording ?

Reply:
ping @Julievrz